authnid/user_repo.py

```diff
@@ -0,0 +1,53 @@
+from psycopg2.extensions import AsIs
+
+class UserRepo():
+    def __init__(self, cursor):
+        self.cursor = cursor
+
+    IS_MATCH = '(%s) = %s'
+
+    def _build_match(self, **kwargs):
+        return self.cursor.mogrify(
+                    self.IS_MATCH,
+                    (AsIs(','.join(kwargs.keys())), tuple(kwargs.values()))
+                ).decode()
+
+    def has_user(self, **kwargs):
+        match = self._build_match(**kwargs)
+        self.cursor.execute("""
+        SELECT CASE WHEN EXISTS (
+            SELECT * FROM users WHERE {}
```

Comment:
`SELECT *` is not safe, and will not pass (should not pass) security review. Remember: Zero Trust. Only select the fields you need. We can always add more.